firstmoversadvantage/fma-ruby-on-rails-template
# 178837538 opt out form

#### What's this PR do?
- Removes all references to privacy apps, and replaces them with simple opt-out form.

#### How should this be manually tested?
- Insall the project (download master.key from keybase - you will need it for captcha keys)
- Use 'Opt Out' link in footer, play with the form, check emails under /letter_opener
- Log in as admin (see seeds) and review the admin panel available under 'Opt Out requests' in footer
- Confirm that there're no references to donotsell website any more
 
#### What are the relevant tickets?
- https://www.pivotaltracker.com/n/projects/2177859/stories/178837538

#### Task completed checklist:

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
